articles/hdinsight/hdinsight-custom-ambari-db.md

```diff
@@ -59,9 +59,12 @@ az deployment group create --name HDInsightAmbariDBDeployment \
 
 > [!WARNING]
 > Please use the following recommended SQL DB and Headnode VM for your HDInsight cluster. Please don't use default Ambari DB (S0) for any production environment. 
->
 
 
+> [!WARNING]
+> Please note that you may be able to change the database password on the Azure SQL Database hosting ambari database; however, it is not supported to change Azure SQL database password when it is attached as ambari metastore to a running HDInsight cluster.
```

Comment:
First sentence says you "may be able to" but then the second sentence says you shouldn't (not supported). Is there a difference in the scenarios, or is it that the GUI lets you do it but that it will break things? Maybe we should state not supported up-front. 

```suggestion
> It is not supported to change the Azure SQL database password when it is attached as an Ambari metastore to a running HDInsight cluster. 
```